core/src/main/scala/sttp/tapir/model/ServerRequest.scala

```diff
@@ -1,9 +1,14 @@
 package sttp.tapir.model
 
-import sttp.model.{QueryParams, RequestMetadata}
+import sttp.model.headers.Accepts
+import sttp.model.{ContentTypeRange, QueryParams, RequestMetadata}
+
 import java.net.InetSocketAddress
+import scala.collection.immutable
 
 trait ServerRequest extends RequestMetadata {
+  lazy val ranges: Either[String, immutable.Seq[ContentTypeRange]] = Accepts.parse(headers)
```

Comment:
I'd like to keep this class as simple as possible, without helper methods

Reply:
(I know there are some from `HasHeaders`, but that's shared among sttp client/tapir :) )